src/pages/Models/Details/generators.js

```diff
@@ -42,7 +42,6 @@ export const relationTableHeaders = [
   { content: "requirer" },
   { content: "interface" },
   { content: "type" },
-  { content: "message" },
```

Comment:
Lets just default this to an empty string if we do not have data to render, as the content may be available.

Reply:
Given we have limited screen real estate, particularly on smaller screens, I don't think we should be rendering columns of whitespace. When we have the data, then we render the column?